frontend-thea-aa/src/routes/detailedContaminate/+page.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+    import QualityIndicator from '$lib/components/QualityIndicator.svelte'
+	import Statistic from '$lib/components/Statistic.svelte'
+    import type { ContaminatedSite, DetailedContaminatedSite } from '$lib/types';
+    import { onMount } from 'svelte'
+    import {ChevronLeft, ChevronRight} from "lucide-svelte/icons"
```

Comment:
eventually I will change these to the other import type which is  
```typescript
import Bell from "lucide-svelte/icons/bell"
```
this is better for tree shaking